indico/web/client/js/react/components/DatePicker.jsx

```diff
@@ -0,0 +1,90 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {Param, Translate} from 'indico/react/i18n';
+
+import 'indico/custom_elements/ind_date_picker';
+
+export default function DatePicker({
+  onChange,
+  value,
+  format = moment.localeData().longDateFormat('L'),
```

Comment:
Does JS evaluate this at calling time or at import time (like Python does)? 

If it's the latter this may be problematic in the future if we end up using this datepicker in room booking, where the moment locale gets set async (since user info is loaded via an API call there).

In that case I'd ather leave it `undefined` here and do something like `format ??= moment.localeData().longDateFormat('L')` in the function body to evaluate it only when this gets actually called.

Reply:
> In that case I'd ather leave it undefined here and do something like format ??= moment.localeData().longDateFormat('L') in the function body to evaluate it only when this gets actually called.

Default parameters are evaluated only if arguments are not passed, so it would effectively be the same thing. Pretty cool, eh?